src/components/Transaction.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react'
 
-const Transaction = () => {
+const Transaction = (props) => {
```

Comment:
this could be more readable by de-structuring the `transaction` from the props.

`const { transaction } = props`